Bottom: 1f92c2979d65c30b77c4d8f2632a59baa9500083
Top:    8bc3e591348e4f85c179146c2f45c8bc31bcc7a1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-07-05 17:01:11 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
